hack/generated/pkg/testcommon/test_context.go

```diff
@@ -229,8 +229,9 @@ var responseHeadersToRemove = []string{
 }
 
 var (
-	dateMatcher   = regexp.MustCompile(`\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(.\d+)?Z`)
-	sshKeyMatcher = regexp.MustCompile("ssh-rsa [0-9a-zA-Z+/=]+")
+	dateMatcher     = regexp.MustCompile(`\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(.\d+)?Z`)
+	sshKeyMatcher   = regexp.MustCompile("ssh-rsa [0-9a-zA-Z+/=]+")
+	passwordMatcher = regexp.MustCompile("pass.*pass")
```

Comment:
Doesn't this need a greedy flag? 
(Though, not sure how to do that in Go)

Reply:
`*` is greedy by default, so I don't think it needs a greedy flag. Do you mean it needs a non-greedy flag - I think that's probably a fair point, I'll add one.